-- Protected by Labs Hub
-- Float Script - Encrypted Version (Fixed)
-- Build: R0LnwytF

local Y2Zpbo5OByJ1={"f3ZhGS4qJz+oNC5PJzcsOS1+CyMtLCo9PnJreygtPCx8Cyk1DD0iJz8sOnNuGio5JTMwKHJs",
    "TDQtOy0ydh00OxEqMiQyOCdsYWksLSwkaB84JQgqIC0oKS5wcwQ6NQ4gOS4mOyR8ckUgOiwu",
    "IXIDKCo+GCc2OTgSOiomODUqf2ZsLSc1OWwFPiQWIyo0MS87fm0UJic9CTwrLjYfOTs9JSIk",
    "cHFXPTQsMzdhHSQqOiU/Oj4ga2VvPyAzPnULMDscKCAgMiwpeWsRIz4qLCgxMjNtdlFGZ2t4",
    "DjMkPiKvKDsrOT9UOiAiNC5vMD46Iic+fHRrHC0gKz0vInUDPTggJhs0MC8qKVcpJDsuNGEu",
    "Ny41MD0IODt2Zm88PSg/KT57CDk5JRAgLRgkIyo8dHQSNzE8IyoFLSV8f0UtOiEuLHI4MyM+",
    "PSo/KTNhb3gtPTo2NylvCSM5IzcsLzg3azc9eDEyOjYpJ2EMJTMkOiw4NDsHKCgkO2IHMD87",
    "d3JGJik7PTpiMyI1ZmViOyQ/JC4iISc9egU6MjYKMzsIJCgtNnB/GS4iMzUuIy8KPjk7Czw3",
    "P3pmUi0xOC4gdS0uLjkmOiwncXRmPCAgJj0iawEuNi8KJTQbND8uP3hnBDkhMzw/NSRjfEhF",
    "bX97GC0iOiAsOTMgta04IlEjPTggJms8NDomLTg3Mgs/PSQ6e3JsZ35FIT01OiNsNyU/Hzwk",
    "OjxwbHZ9bVEgJSU5MHY2OiIiIywWNyMyGC4sMGJyYHAYNCsgfAokISMucFJXfHZvBDozI6ou",
    "ND88UTEqKDkjeCgtFiA6PCEobW92PS4gIixMICMiPjRwMiM9LT4iPgU3MjgnOCQsKTZINCM9",
    "NyNhMy42KTw8e39sOignPyRLPjc+MDdvPSkoLSI2MDoIKTwzIiw2eHx+NSYgXyMgLjM6ey4i",
    "JSABPi03NiwpEjkhMT4vPi83MnZ/ez4sKlJIdWF+a3J8aH9vBic1vLcpL2kPKS0oJD0vKHsb",
    "Ant8d3ZlbFwjND4kJ3gpLS89LyYjO28pJDwyHyogOD8jPjUJNiwyPi5ndlFsamZ4MDkhOjxl",
    "NjQtLD9+a28WOjAkMyI6OCd2GiAlKAcoICspEjMmPj9paBs0PiI8eXRPa3hveCg4eyEjIW8/",
    "IT0iKG84OSwobD4kKy0iP3YhNjdsLyg8VnZie3AjKSpiB2B+JiMuIWImLnIyKyMlLjpjPC0u",
    "JitnFj47ETMoJi8qNDhncnRlLzdFeGF+e29sdW8jIjE3N28/OC4obHFhLzQ/JWwJNjUoDC8q",
    "LyIBMzkpInBgCCAxIhwoMixtaVh7e2JsfGlrbCgncis0NjVvMzUlajgxNjh1HTQrLx4mKjIq",
    "GCclOStnbws5Lj0OMDojbmVhPjY0cSUmODViEyktLhQjKDVrAzYjOiA7Mm81PSchSnJ7e2Js",
    "fGlrbGFhcjQyMjojcj8kJiIuNCQ2e2BlOzQgLHsYMiEoEyY9PiYVMyYgNWFkCCktNi41Iy8H",
    "Ni8uJT56dVxie3BlZnhieGx+dm8oM2IrJT4yLSc+JWk/JCQvWHh9cXtvcnthamt4cXZve303",
    "Liw6Ki9+PyogPDkqPyt4CyA/OD0vIyJLf3hwcXZvf3tsamZ4OTgmUXBlZnhieGx+MyElX2Jv",
    "YHI+NSZGfGlrbDMkJi0vP3shOzdLLyU8W1wjND4kJ3gpLS89LyYjO288OT0mFiA6NCQjIjhp",
    "dlJwcXZvNigBJTAxMjFiZnAjJzQxPUZ+dm9hPCRvIycpKSciKAokIi8kMSw0PjVvJjMkJEF4",
    "cXZve31lazs6KjM7NTsPOiEhKDEiMiAiaw0vPy8uMTY1MiJndlFsamZ4fHZiezMwNConNjgd",
    "OSEvMCE7KT01e39sMiAnRmFhcng4Pz9FcnthaiI+cTchLzQCOTk5MTUnGCAiOyosOTs5NW84",
    "OSwoRmxhf3hwcXZvPjU4IwEqPSArLykGKTYsPS8qPyAvbwYmMzE0NSwpPz1jZUthcnh9cXtv",
    "cjovPiIfIzc5Mik8CDchNiQ9LyYjO29ybTw/N0VscWlmKSIlVXhwcXZFf3tsamt1fAQnLTU3",
    "MmJiPCk3Li5hMjAuNjs/OiYpfCckPiwgPng7JDUsOzQvKzlScXZvezU3O3YOKzI7Ni0gLAMm",
    "Izc3KRkpPSYlJTg4f2VwBzMsKzQ+eWg2OSFqa3x1amhrUmx+dm81NDEkbiU6MjZkbGd6ZWFs",
    "f3gNNCo6NzUuai89PTc2ey0kOTlvPyAsOiE4PD1vInIkPjkpIz1MbGxhfzAiIXgOLCgpJyQ0",
    "JRorNTUkNA4nNCM9Pzs4dX9vFjc4Ly0+b2clKTZpYnRwZGtjYnJhZ2Z4Fzk9vDxlOi0qPCB+",
    "NS44ID0uIVgzNStGWyUpLy0tfz4lPzU7NjQiajUsPSQ2Fj8zIzUnNjh2f0VhdWJvKTR7MjEB",
    "Mz8iIiZhJjA4P3s4MykvYmkSsHYqKCmkaz0ieCwxLSYhMCE7InN0cm8+ND0zPiJhOjY0W3Zv",
    "f3sgJSU5MHYmPjwsMD0wIRwxJW98dSQmLjYfPi4lKiw5NQkoJjoyKXNmWHthamsxN3YhNCll",
    "Lz0jMTc7KTYcOjxvOTozNW87MDsoZG4FOjQ5JzM9JhMlPiQ3JHYsuD9lIzYhNyIqJC4lNGNt",
    "aXIpPjY5LidrKS8lWHh9cXsmIRYuPCI2NnZyeyk3Pj1FUmF+e29heG8cLD4gOj1sNjsnOiUl",
    "Pjw1cTk9NjwlJCc0VnZie3AsIHgsNzh+OT0oMishIT4cKSM6NT0ybDUpNzZXcXtvcnthams3",
    "Iz8oMjMkJx89OTc3LzZsaG8YIiA9KD8tMixoCz4gKTEkKFxvf3tsLyg8Vlxie3Bla3ViGSIq",
    "P2ImJyM5KTY6PydsPyYlPzUgPCw4cS48MzUlJWsZIiUqNj8pMhQmNiQ/KRkpOSAsJCYvUW9s",
    "cWknIjgoGCoxJz87JhgjJCg9PyIrND5le3gQLSINMz03PCEqbho+OjA4PiwqOHsCPTYzNDg7",
    "ej00JCgsODkhc3RPa3hveGF+e28lM28hIiZ2MjwBPj8vIithMCpwPzk7fzM+OmY3LnYsNCRl",
    "Lioydhw/JCovIWI7KDc1ezApKDw5ImEkPDxXcXtvcnthamsxN3YpNyQsJT9vLCk7NW8+MDs6",
    "Pzx2PiEocWRrbAKiMHg5PyIqLT0pOCN4LSMjNTQqZj4uIWw7JTugdSM7KSQ0UWJsfGlrbGFh",
    "WHh9cXtvcnthZ2Z4EjchODgpKngoKiAoMistMSpvOzckLyYvMCVmLyMvLCwxPyIqMj4iPiNS",
    "fHZie3BlZnguNy8/Om8iIDA9JTwvDScgfHRrJDMxfBkuIj4iMDc4BiI2NDc9DTgpJDsmLDhU",
    "e29sdW9vbXI+KT9iEDo1KSEjMyEcODgqPikaLyo3Pz82InB4Zg4nOzgxJHxvOyc4aDEuKTAp",
    "Mj0dKS1vCnR9YXdvMS4zOC42JQAqN3MfYlJveGF+PiEofEVFbXJ2e2JhcQUpIzxhOz1wPDk5",
    "NjYpJDI3fCArOnAENSsnNS4yLwMoOycuMgQ+Ny0vNT0yRmFhcng+JCk9NzU1CSQ2PzMsLzQq",
    "JXhyeBMrNRwpJzkmLjd4EyotIz0kKS01ZRs/PzgqPC9kLDM2PyIrND5tb1JieGx+dm9hdSsp",
    "YDw0L2IlLwQkOigvNXgyI3shPS9hIjkocTk9ezMqP3gnKjFwCy4+MCE7bSY+PiFsIj0pPAEu",
    "KT09NDg7d3JsOCMsKSQsezUrIlJIeGx+dm9hdWJibXIfMjApu6okbCkuIDEnPjU7MzdhOioq",
    "MHYLPjEsPT09IQk3Ly0jLUVvbXJ2e29scSUpLy0tfzslIyQqMS8cJTV4YXYqKSBrFjcxMTg3",
    "OSFLdWJvYHJ7e2IgMyoqIGElOyo4Mi8mPTVhd2sONDU7NC92ZTYqL2k6PiMlIyo9NAI5KGEU",
    "cWRmLzkzLT0+JQYgLHUUZmZocHYmPjwsMD0wIRwxJWEbdW9vIycpKSciKBkkP28be1J9cXtv",
    "cnthaic3MjcjezUqOTE1Ny8qOiMIPDw7LDw1Pm9xcS0vPikiKzE/P3gCPjwiIzItODNIUXBl",
    "ZnhieGx+PylhPS09KSg0NTYtMA0iPzUgPDs4cWdycmhhPiM9P1xve31la3hveGF+e28/ISA/",
    "AD0gPiIpPz1uZUZhf3hwcXZvf3tsamYqOSI3KT5PZnhieGx+dm8kOyZFSnJ7e2JsfGlrYWxh",
    "HzcrODYqPC8uaiM3Iz81NDMxKjRvc2E/NTsleCg9LCQ/Py4oNENmbGxhf3hwcTogPDogajA9",
    "MDkhMiQ8ZmViPCUsMyw1PC0hbgc1MjZsdmkvKS0oJD0vKAg/Nz4lQGt4cXZve31lPT0jNyI3",
    "LzZsaG8ZKDEiND1/fycjO2Q3OjQ/Mj87JnUUZmZocHY0PjwqJTE2IWIEf29seGIJLyC8OmIV",
    "fHRrfGExMyo8cTohJjJsLTk5Jz8rOjkgQXhveGF+e29sX29vbXJ2e29sfGRmHylhMHgWPS9v",
    "Oig4IzA9LnYjLzkzKXRiKCksOyY1MGIiLyQyNiciKCZrOiQzJjE+MDdFcnthamt4cXYmPX0j",
    "JyEmNiZ+LycpO0VvbXJ2e29scWlmbGwtMDsxPXYpMyIaLyp4YXYqKSBrBysxPSE8OjYNPCwq",
    "ISANPi4jPyA/NUthcnh9cXtvcnthamsuNDogODQxMnhyeBc7ODsjJ3xhIzchczkpPSYlJTg4",
    "cQB8cTAjJg0pJmgBcHY0PjwqJTE2IWIEf0VhdWJvYHJ7eyciOENrbGFhcnh9cVFvcnthamt4",
    "cT49K3MEOCsqNSMyIgMlOyouPwQzNyAvOD0/bHFhKT08PjUmKyJGamZ4fDMsP3lPIzYmUkZz",
    "e298aH9yfXIcDgtsGCwnJTckICF9BQtvb2Z8d3ZSPTksOjFlHy8qPS8NPj06PCwqbW92PC4h",
    "NHMBKTgSOiomODUqd3kYPSM9MgUnKSYsJT1gcUZUe2JhFi0hJjs8LjAtu7wuP2ElN3g+Pikq",
    "IXssJS89IzguKFcpJDsuNGEdFAMDBxxvcHItUW9scWkWHgUMHgoJcWtvHDQgJTRrcjAwND0X",
    "ARpqYXVydn5xZ25vcmZqcm5scWRrBS8lOz8yW3tvcnsSDwgXHxIOCQRldngMNy0xKXxiMz0g",
    "IAARGWd9YnBqbHVzc3hiZWBmc3thZ2YIKSQyNzVPZnhieB8LFQwEBhFvfXIYNC4jLnplKjMu",
    "PwoaE3N8Zndhe3JvfXZ2b3Rpa3VieAYsPioiX29vbXISGgELFBtmcWwCMDQ/I2VhOSkjJxQf",
    "Hn5waGlpZm56dGxobmZtdW9iYAA+P0hsfGlrDgACGR8PHg4BFnt8agg3PTk9aHMjOTciCgYc",
    "c355eW99fn52b31lfWlrYWwSMzkkNHZ2b2tGamZ4fAUXCRYEBR1iZWwdOSMuJ3FhJiA0NhAL",
    "HmF4fG1hZmlxcW52e3dhZ2Z4AjouLzhlc2h/UmF+e28PFB0LbW92GCAgPjt1YiozMDUCFhRn",
    "ampganBtcHZ6bnlpZnVveB8yNzskdXV/cFh7e2JsCAwTGB4RABEQEAkWcmZhCSQ0PiR8dTs3",
    "JDUdHwN2aXt0eW99eGJ6e315Y2BqbGFsfws8MCIqf258QGZ4fHYWHggRGQsHGwMQEg4TDGJy",
    "YBE0Ny0+b2ctPi4sAB8feWp7andhe31rfXZ+Y2lsZ3hidWENNy44MG97fWJce29scQsJHggE",
    "DXhtcRUgMzQ+eWg+LjkvCRcHbm9zdGxmY2NhZHJ6aXJ2dmIfMCg/KWF3YmhXLFFFPjQiKyd4",
    "IjU9PjgrDC0meHx+EiE/IS4hLjd4NSo7eWsVLz4kOjYXJD9tdlE/KTQ9OTgFLjlrCDkvPWxj",
    "dm0FMC4mNjcpIhYcGzwibksyMSo4NDUIJzJvGC4rNCIANQ41Ki8heHx+PS4gJipFPjEkPioi",
    "FjwvYggoLCg8MC8ALT8pOGZlfG97Ylo2JSonPSIZIyZvBSM9JTwve39sLCUqNSQzFS00W1Fi",
    "f3sHOCo1NHY/KTQrKDE/OS1+OCAhdSgjLCElNiA+ISEvPyFLMzczMDpvMjolJAAqPTsne21l",
    "DzYxLC0wNSpvOyc4aHAdKSMhOWtiRiwgOzYbIzoiN3USIzE9cWtvDhksJmphNiQpc39gdX1/",
    "fX52a2NsaXlvRiEgNjYWIzciOnUcJTUxKD8tNXB4Zg0GMSFseCEkImp/bmd3e299bHlnbHFt",
    "cmpteFEiMzIvDDk5PDNhGTwmID89NzQwPwwjOSA9fnJrewwDHQYUH2ISCgoWEBUKVTYtIyge",
    "LjcvPn4HJzspPz4xIyElATAuLiErOjApMioybHxhYnZsWzYuOzUHOCo1NHgNNC8hLiocMTs7",
    "CyY0MCNvcHJmUSItOCcAPi0sOnYRMiImKT5sd2YsLiMnUT0kLzYEKi0zM2EFJyMoJzM5Nyds",
    "YWk/PjQkWDU8ODUJIDosL2UIMCQqNSlldng8OzM7PiELICZFR397exwjPCs0LWwyKiw5PVwj",
    "MDgtJmYrNDcmNCdle3gLNj8qNyEiMGwhJSVzeQQ+PSQubmhLITA8NTQ4fAgoMC54bHYaHzQo",
    "eXYhPTZ2amNsbWNvfH52Y2ZGIiEnKCM2cQg/Ij87NjQiant4CRIrNmJrKD01cHxydmJ1eWJ/",
    "bHJ2b2tGLyEqKC42fBo8MjAoIDQ0JC8bPjogKW5ldngMNy0xKXxiMz0gIAARGWd8fWl2YGxx",
    "dlIjOTcrMCxiCCc7NzEwNCUrIgwwOSItJi4zMCwsOXJme3Jia0M4JCAlPS9zCxIhNj45anZ4",
    "fGdFKDUkLzc4dhE/KSoiIW9ybT83MiEKIygrKUZLMzczMDpvLDMtLikvHzkwNTU3ZmViESIt",
    "Ii4vNidhLjcsc2AZFQokPi8kIHp0WygnMz8uPQg3IzgqKXMGJCohPTMMOislIDxvcHIDHyYh",
    "fycjO2Rxc3hhZ39FLDMtLikvHzkwNTU3aAgjKikwIm98dTEnITY0LEhGcWRrDyAvJjcucSg6",
    "My0kOUE0PjUuN30mJCohPTN+Zm8FOzw7LDw1PmEiND5ubhkIHDciPzM9fXJGKSkqMjMwdRMq",
    "NDYnKh4/MiY0JmJyYAcfMi9iMiw8ZHFtcmlreFEsPSkvLzl2ATc9PjMxa2VvNSA3NQk+NCIq",
    "R1h7dm8OPjsiLWwmLTk0ODMhKz5GJik7PTpiOT83Ij0weHF+HyEyISMhIzd1NSc7dGsNPiAs",
    "N3p0WzkgID8kOGULOCwqe2BlHhwmNXNwNSo7fX5jbWJ6e35gcXlvRi4uLTw1I3gfMCglPi83",
    "MnZ/ewUBLzVwdiI7IWdxeWJ/bHJrd2J8dUMpIzMlNypzEzosOTwzJT42NRUgNzI3eHhyeAIR",
    "FwAeBmENAgASHh1GMyY0KCkzcRoxMj0oLTQ5JCIMLjcsKCAkND0sOzV+a29xe3FFIj0pPyc+",
    "cgskPiUkIAs0Kz4fOyMkJmtlcWZFOTI3Lz09dhsXNSspLW9ybWNcOSA+NSw0YhwgLT0+JXZy",
    "fzYtIygeLjcvPlpPKjchOSB+NCAzMSc9Az0pNSc+fHRrBS8yJjkzMj5hPD42YmkNGBUgKTMg",
    "OXpmUiMxKSspJwwgPzwzKWEPPjsoKT4TPjw5JCVvYnsZDi81cjgnLHh1anhzbmVUNCAzMSc9",
    "Az0pNSc+chkqPiQvJnhgcTkgID8kOEFSfHtvEzgkLz09eCwxPyo+OyBFIT01OiNsOSwnKCkz",
    "f2VwGDg8KzoiKSN2MjM1c3IDNDkvPW53XCckNCYqMnwIMjgpfHRrGQUoP2pzPz44emptant0",
    "cWZje251YlInPSA6Pj1iBSA8JCY/NCFsbGkTCCUsbXY+NCFnb3dsemp4bHpia3lPLj0jPCks",
    "eA0gNikoMj0uNSYPMyUkPnJhb3geHhcAAAhvCQoKFVwnPjwhLiphGiA9MCg+OjohKQYkOiE/",
    "ISg0KSIiJnhtcWZhbVEkLyc8OSRsGT83Ij0wCyUkMx8oLScjYG97a0gkOSgvKTNvCBEzNT43",
    "cmZheEEwNDcrPi9rGzk9PS8qe3JsOC4mIxQkOiIpW0MqIy8gM3g4NDcrOikPJTQ2OSRiZnAM",
    "KCs2OSI9M2EvMDVnYgcSGC0+Miw5bmhLOj08NT49ETQzJC4qfxUgKTMgOQouPCgrKG9xdRoL",
    "JD94NSo7eXlqbH13dlI4NDcrOikPJTQ2OSRsCzE3IzY2eHF+PiogMSc9Slh2dmILLigvJSQv",
    "Jj19PzRvOj4gLi4qWzogODwpazAqOSU7KQg+NCsmKDwie3JsGCc1OC0vPD1+PzM4d3kZAwEq",
    "PTIrPj4xZHFIMCk/MiozEjAuJDs+NTZiHyYnIzNhb3gePjcgIAgkOz49PzUqdTMgPCNFeGF+",
    "ewwjOSA9HjcnLioiMiwNKTUxMDE+JXghOixkemp4HxkOFAIWaAgQEQEfBBZoeUhvYHJ7GC0g",
    "MzsYKTA0NzY+NBAqKysuIyUsfzgqLHV0Z3gMFw0RCRxiBgoMAhwSGh0VeEM7RiQkPjw1IxE9",
    "Pj8lLygscgIwOj42NjkwPSI9L298dQw6LTA+KREpLTwuIiIkfDY4JiBFcnthagUtPDQqKQ4g",
    "Oi0qNiI7ECo1JSAmIyZ4NSo7eXlqbHxvZ3F8W3Zvf3sCPys6OSQRPiEwIzYhPQc7Lz8uPCw7",
    "bjw+LGp9cGl7YnhoWCVXOT4uNj4zDTk5NT8qNSlrGzk9PS8qe3JsPSouKTckUUVhfGmLLyMv",
    "OlI8PjUuM3slKSk2fGtiEj42MjksOylwOCo2fWAbJSovFyMuOSVpZUsoMTczfwgmKD5hd2sN",
    "FT8iaXMrLi9naG1+aX9gdX9jbWBmckUlMiYoYhwuLDEkODkhf2ZsHwIxMWRsNTUybmhueHRy",
    "dn9tdXdmSjs4NCxiHigoJyYzPS0zNQ89MzUyOioqNDgsIn14a2lFMSIxNWEYMDc7bW92eW1G",
    "OCopImIVOiAkEjkjMCl/ant4HxkOFAIWaAwHABgBBh0IGAMdGVgyOC0icg8kIjVhb3gYPy4i",
    "fB0uJD92Fjk7MzwoCTcjPEs3OCAiexsqNSYFMjUpcXRmfX5LNjs/P3gbOiM4Egc0NTEsNjUr",
    "Mnh/eAkwIyJvASc3NAoaNysrMiQuIjVvET0zJT49WDIiJSV2BTM3LwQEJzEoNiw7NTtsaG8K",
    "Iyc7dRspKT0fDSAoODY9NDg7cRgpJDI9LlwrOD8raAgjKikwIm98dSoqITY+KUhGcWRrGKw1",
    "JzQyWzcgMTotaj8xJToqe2BlAjY8LCAwOCpiOyo4ZXACPjc4HSgkKSBjdlIkOCIjOnUfIzw9",
    "fGtiDhQsK2psNikpfn5tdW97cH57a25sbnliRjUoJjQ4fwsgITI1IyQ2cWtvDhksJmphNiQp",
    "c39gdXx9YXJmd29+eEMyJTgtOnYSMDUkOCkjPyg8CCQjNSM1JyonNi8ndnJhZEg7KSY3PmwY",
    "OTE/bHxhcBQ8MyhvGi4jamZ4FzogOilnQSwmLC07dRspLTsMIj45KXxsbGkFAwAODQt+BRMX",
    "CwQcGA8VHQQbUSQsMjQndgoxODthaGIKLic2dQQjMj1lCy41OjkwEzQjNlE1Iz80NHgbPiUx",
    "GDE1PWFje35+XzsmOT4zdRspKT0eDSAoODY9NDg7f2ZsDygtMXgWPigxHhkuMSswOyovIWwD",
    "JTQvUTYlKCUuYhUkKiwEEDcmNTUsLyUscWtvHjMwJnYbPTkqAg4gPCghIDc4L2EPNCcyKT5L",
    "KzEkPTNhDzo+LygsfGtiMzUkIj0wUkZze28DOjasL3IILyM+KEMnIyIgPnguJTo9Jhk0Pj83",
    "P3ZyexQrOCwuNiI7dSEpImdtGTcuLw05JT0pIm5oVSskMCQ7HS44Pik2cgUrITVle3gXHCUz",
    "ZGEvMDVncHxvbm5scXxnbHFtcmpoeFE8JjozPgktJSIgNXMVJCsmLCgxNW9xdRoLJD9kdSEp",
    "JmF2YnxzanRwYXpvb3V4ZmZodVwxLzE3Mho3LDgxOGEDNCEkJyA0LiwoHyYnIzNycmV9EhQD",
    "HQkSZBgNEhUKCA5POCwuKjUcLjs4OiFhDz0kPyo+AiA8KRwoJz08cWtvb1E/PicqKBQ3LyQq",
    "KHYWPTQqdnJhdxEbAQAPeUg/KCg5OAM0JiwyP3UbNyM1CSQ0PiR8e2BlCBcDFxMNdRsJDRsQ",
    "HQAfFg4eCEM1OC0zKxolJSIgMXUKJSgsfGtiHj4wK3YENyIqeAguISouLRA0NyZGLz0qPjUD",
    "JywpPjVhBj45PhgxKzNvZn10e1I8LCAsLw05ITsgI3wGOj0pPz1mcWwsPjE+FyQuMj5GOTI5",
    "LiIALiQxKTZsFSMrJSoDIDY7LzxqGC4lPyJxDy4vPD0+JXM8JjozPgY3JzMiPjMxYlJFNC49",
    "OiNsJjsuPyYVND0iNDtmcWwIMSskMDgsOnUiLzFwfgMLGD83KD0wemVUJTsgJzYMLyA1PjBi",
    "HyY5IiQzADk5OC48cmZhHw8xPHghPipte3RvYGhUKDstJzsMIiA4Pj1iASg0KSI1f2VwIiIu",
    "LS8OPzIsMzhIUX1oZhotLK8xdhw1OjJFLD04Oi5sLz0kPAM0JiwyP3tychIvOT85PzUqdTMg",
    "PHBtDCQmLw05ITsgI3B/UTw4PjkEOTg1MDZ+Aj81OntxahMcNTtwdT4gMXBydnhrem9sYG5v",
    "cH57aXdlVjo/IzEDJywpPjVhAjQyIz8xPjhvZn0QDzEiam8wPjhkZWF6f2d6e39gcXloeGBh",
    "b3FaIiIgLxk5PjI3MngAOjMuISotLSI6FSAtOjB8YG97GA0AExsYYgUAHB8YA1E8JjQxCD4s",
    "JTkhdR8qOTwqKhI3ISocPDcqIXJre39GIj0pPA40Kyw/P3gbOiM4ant4fgUWFABnTCs2Nzwc",
    "Izs1OixhFDcjLwEjMCY5f2F8chsSHRQdAXUVDxMMDgYdEhAEGQFFKzUxKw05ITsgI3wQNCE4",
    "cXRmCSI0MnYWPjg7cRwjPi45MRQtNzRPNSwtKA4rIjsuO2wbJSovCCs2OWl2bHBxWCspPisN",
    "Jy81JSV2ATc9PjMxa2VvNSA3NQk+NCIqRyEiND8OJD0yIyJvEjclIjMNKi84JShpHzorODt/",
    "BTcsNik9ImcyIS0/DT0tPi8pMj1iRkstPTs8PXs8JjQxCSQqPzM9e2BlAjY8LCAwOCpiOyo4",
    "ZXADEgwjIycjPm5oVSskPiYMMCkiLzR2HzkwNTU3FDkmMTktdnJhAAYmLXw1PjVkbGVrdGhL",
    "ISwyIRggIDUkOGUIMCQqNSlldng8LC4uGTo4ISAhR1h7dm8FPy8pbCgufx48KFwjMDgtJmY+",
    "MC8OOjIgKnh/eAUwJTsgOyEqbjw+LGpuCCwzOA0gMD0xc3JFNDc4Bio6NDphCDQ/LnhyeBQa",
    "MiJ+eyEqOnpnd29hYHlqbHxtf2lleFwpMyIAKyQ9MHgSNCMsMjEtNmxjdhoFPC99bjw+LGp8",
    "cGl+YGFxfG9ofXt/e1EnJjIUMDQqN3MHKjskPzMxLiEoAT0uIyEmOj0pPyo/bHFhblI2PS8D",
    "PjkpJmgMOS42e21lZB4uIXZ+EG8qMDttSjQ3Ig4tPiwnYhUkKiwePjcgIGhhd2sbHhoACQ5r",
    "Hx0XDB4NHgwDGwsOHwtcPSM1HSgkKSBvGTc+JXZyfx4iPyt2GjksL34CKSwqOSFUMCM4GSMt",
    "JT51Dyc0KBoiNiRhb3hkWz0jKxcgKC40fwIqIykdCjQmPy8zPiE4dXJvCDwjNmEYNDEyFA0t",
    "Nj8+PDMhK3UPLygsOSRIPTw8CjkgPSBwAio5IRsOLDs8NS8pMj1rcWEEPC0wfw8qKi8YCycx",
    "NjgiPjMxZRsqNjU7KUUqOTYDLDAzN2EcMDsjIjhhYng9MD8hGSktJyNSVntvexgqMD0weCk4",
    "MCoiITFFMyY6KTYOKT0/Iy9vHzcoIj4KPC8kOHEbPjghPj4xYz46NiIqMiAifWZFbXJ2eyMj",
    "MigqbCQuKT0iBSEqOjVsd2YMKzMnNQMgNC4rOylkFT0kNDYqaCEvOjA4Hjw/OC4vfngJJj4q",
    "PBIvLCR2PzM4c21reXRvHS8rNmEJNDwmIzUFLzYgNGcXOS0ldnRwKlxvf3tsamZ4fBQjODsi",
    "NDc3NigdOSMuJ3FvfXIYNC4jLnplKjMuPwoaE3N9YHdhe31rfXZ4b3RPa3hveDx3UW9sdW8n",
    "IiQzKRs7NCwodhwtPiF4eFwqMT9lQDUsPSQ2GSUxMjcsdgExIzwkGScuNjdhGC0iMiwoOGkn",
    "JzY+JTIgPHNoQGt4cXYjND4kJ3gnNzc7KRs7MCohbW92DzgpNCcVKT43Njs1axU9Ojo4L24r",
    "KDcwLxIwMiwtNmB+AjgkMCwGLjQ0dSwpK2F7YnNtch0zJDZhFzoyIyU/AiI2NzhrGi0uPGhy",
    "ezRGdW9vbXJ2e28OMCotKz4uKjY0EjkjMCl/ant4HxkOFAIWaAsXGw8bBRxLdWJvYC9yUWJs",
    "fGkjIzckIAwqND4haAstKzJweFwqNTlsQVI8LC4uGTo4ISAhYx85LjwpFCcyKT57HDc+PzMs",
    "K3MqPyg7KD8tNXhsTHhieGwyOSwgOWInLyQ+KRY7OSwlbHxhBi84NDUcNyk3Iyg9axU9Pjwx",
    "LnA8LC4uGTo4ISAhYXICLCopPwAoKiNvMT0neWZhbXdsDygtMXgHOiMsKD8RLDUyM2EQICMr",
    "aX57IEhsfGlrbGFhcho8MjAoIDQ0JC8bPjogKW5ldngMNy0xKXxiMz0gIAARGWd9aXxqbH55",
    "c3hiaX9Ff3tsajtxVnZie3AtKS4nKhgpMyovbxIjIStzckgpMi1iRjI1PSgfJC87PTVvByQt",
    "IjMDPjwzLmIMNy8wPiw4fSk6IzEiMiAieWBMbGxhfzQ/MjcjfzMjPCMqCCEnPj5le3gWLyk7",
    "OBwkJzQmIzdhGDApPT0uZDI1PSgfJC87PTVtah8vNDMhEjMjJHYhPTZ2a2F+eW8KIyc7dQot",
    "IiAoKx81JjQ1fwc6Pj9lZmYjVnZie3BlZnhiGi09PSgzOjchJBE0Ny0+b2l2bAIOHhcPAnUL",
    "ExUGDxlScXZveyBsQXhveGE2NDkpJxs4KDc4YR8gMDBuZUYkMTx5W1xicnsYJTQ2PSRiOiI3",
    "Jys2uTo7OkUtOiEuLHI/KSMrOyAlK21hNio8Ngg7Myk1ZmsrJTc9Lw0qOFIiOSgwHT0tOCph",
    "BDwmLjsONC4nInYCMDY+NDU7dz05JCUsNTksczkrNi02cUZ+dm9hPCRvKTwrLjZiCTouPggv",
    "Ii0pBSI/N3t8d2sdPyMidQg2LioGNjErLxs1JSphAD0jKCoOJD0yIyJwfyw4NDhFf3tsamZ4",
    "fHYmKTEiITEsP2xjdjszICdFYHJ7e2JsfGkvPiAmASw8Iy9vb3soJDstJXgfNC4sPzEgNkt+",
    "e29sdW9vbSEiOj04ASY1bHFhMjk5PxA9PjYpZBY3Lz82Mj8rTHhieGx+dm9hPCw/NSZ1GCot",
    "Mi4uKHsCPTYzNDg7ej00JCgsODkhc3RPa3hveGF+e29sdW9vJDR2MiE8JD1oGT8kLRE+ISM7",
    "DC8tPiN4YWtiHj4wK3YXKyksHyExIDYcNDMvPmwJMi1rOCkkPHg5IzooNTIvLWtlcTAuNy4g",
    "az0hPEt+e29sdW9vbTc4P2ZGcWlmbCkvO1I1PzJmVTYtIygeLjcvPn4MKCg3LA82NyEmMCZ1",
    "Az01NScvKGEtOS8iJjEyP3MmPCs0PmJScXZvezQjazEhKDQqdRo/MD0GIyIjLxs1ISxmcXFh",
    "GjYlPHgaLD4+AygoKSIWIiAgaBUtLT87GyA3MC8qLiZ7OiwofC05LSYmOzY6cS8nNzVLamt4",
    "cXZve30pJDsuNGE6PiM4NG9ybTs4Kzo4fxkpPyU1Njc+cXtvOyktLRUsPSQ2UXBlZnhieGx+",
    "Oy4oOwQ9IT8+dRIjLyA/JS4vcmV9BB8mP2lvJC4veSU7Oi8xGzc8dhlwCCwtOSpjbSEiOj04",
    "ASY1YhRvED42IjM7f3BsLiM0KDdsA3xlNSwjKjgOOTxvDGwcIzM3Pm5sLz0qPjURPStzCHUA",
    "ND0yLz94enYrPjExKnYWcUt+e29sMCErRzc4P2ZGW2RrbHF8YmVtcRAjJntxd3tlYVwuNDMk",
    "KngkLSI9IiYuO2I7LzU8NycKMDBjZUthcnh9Nzc2OzUmanZ4Pzk7ezspMjEhP0t+e29sPClv",
    "Iz0ieykgKCAoK2w1Nz0+cT49L3UNOTU9MTQuIhwsKD0jKho7OiAiPDY2YG97DScvKCY5f28v",
    "Ny91YXd/fmtoai42NVwqNTlPQQ08PTMXNT85IRwqPyQ/OCpiGCc2OTgDOj8xP2wMMDUiLyUs",
    "dDA3NTMxLzcscCUwJjo1eWIoMHtRe2JsfCAtbCYxciw1NDVvID41Pzk2cTMhP1dla3hvMSd+",
    "MiE8IDthBjcvGCAoNGl7cWwEMS09fx0qJhgjLiN2GnY2MzUrZiwtPysyMwktLGpmYDc1P0gp",
    "Mi1iRksTJzYONCk5OzgkZBk9PzIqKQ4xLig/PSVkGCAiOyosOXowLiEvJSApImRoVXhwcXYm",
    "OXsqJj8xMjFiLzggKFJieGx+dm9hdS4gIzM3ey8jKiwPJTNhb3gLNDg7PSlyZCU9Jn5mUX1l",
    "a3hveGF+NyAvNCNvLjM7e3JsBiY0Jz8xPjs1fxU6LSkpJDIbPTsnKTFrBR4wOSE7XG9hdWJv",
    "YHJ7Ny0vPSVrJC4zOyIyPy8uPhcuJSB4bHYZPj4xJCp8di87LGcvNCJhAT05MBkpMj0pPmIZ",
    "c2h8MjcicRcjJS0OOTU2NCJrHHFsDSI3IkVhdWJvYHJ7ey4jPygnbCkuIDEnPjU7MzcTIyww",
    "JXZyewsgKCwgKnJwNSo7fSwuIHwEMigkJR8jLzguLXYIfWZjPDohZBQxOz42DTUmMjcwdhZ3",
    "eBovPDZFSnJ7e2JsfGlrJSdhBys4IxIhIi41GS4qJz8sPmcMOBMqIQUxLCFkECE6IHwdPjYP",
    "Pi0jYhtofyw4NDhvMjQ6LwIxLnZpZnAtKSorIiMwIi4tGS0gK3I+NSZGfGlrbGFhcng0N3sa",
    "IT4zAyUoJCIcPi8zIjsqYggtECo1ESA4I3oTNTohfwIjNQ8uOz1+An9vKzMpJGY1MyAnHzk3",
    "ZnV/eCQxJCY7Oiw7IT4XNC0nfCwlKEthcnh9cXtvcjInah4rNCQGNS0wPwsqKjc3OCp2HDwE",
    "KCsSNDgieQwoOSFvFD0pEjkrOnUNY2YsNDMsez0qMD0GMT5+e3JhPS09KSg0NTYtMBsiKyk1",
    "cj0zNVFvcnthamt4cT8pewg2LioGNjErLxwpJzkmLjdsEjwHNDACIzsvdx0+JDthFD41CSk8",
    "OXgGcnAxLj0seCExICoFPDBva297My0+NTMkIjUgPgo0NjM7cj4vLkF4cXZve31lazEpeBQt",
    "Pj0FOz86OQEzKTklMix8BT8KOiEUPiEhdx4iPyt2FzM7GD8hI3YRKC09M2ZhISoqLnI2NDQp",
    "GCA5bGp8cg44Mi8gIGhvJC4veWZjanF1YngqNiVUe29sdW9vbXI/PW8ZIiw0BSIxKiwDNCQ5",
    "NjgpcA8rFzM7Hz8yKHAHNjkzeAQkLAEgJDd1FycqKBojJSc1e3gpOT4hcjYuPC4cOCRvdmBl",
    "HT0sLC4saGEiMDhnfX5nd39lcSwoKEZLf3hwcXZvf3skODZ2HSUxPj0nKiEOMSI7Nz0XMC4g",
    "IzsvImJxfCQkOiQFOypzHDooPDI1Py89cWhva30kJTxvNS4oPgslJ2EaIzsie2VsNyU/Hzwk",
    "OjxwPiRvCT4vPikqb3gsPidtdnRydHx3XG9hdWIqLjZRPiwodUNBYWxhb2VgbGZvEy80Kycx",
    "Kzc9ey8gLT09si89Mi4/dSEgbSAzKD8tJidmcXF8YmVaITouJj4+ZAUwPSQjOCQgNBkmPCk6",
    "bAwuOywqIyZzPTciPz0iIy9pMTA8I3JFcnthaigwMCQuOCkgOXhyeCI2Oj1GdW9vbTokK29x",
    "cSouLT4gPCw1I2wYPjI4DCkqHz4rNzRtZBA3NS0wOSYlBy0gNAI6KTZudUNrbGFhMDk+Oisu",
    "MTBhd2soPTc2Pi9/HDkmLAcxKQwkPCMrZXAUOiwnISglJ25oVXhwcXZFf3tsamt1fAQnKDUx",
    "Zi4jKiW/ICooJmIhL3IpPjE8PT4lRmFhcng0IhYgJDIvLWtlcTAuNy4gQXhveGE4NzYlOyhv",
    "cHIwOiM/NENmbGxhNj5wMiM9LT4iPgU3MjgnOCQsKTZiLCQ7OEVhdWJvYHJ7eyE5LjsuIjUC",
    "PTYzNDg7OzQvcA8xIjUgNTMgKCxncUt+e29sdW9vbTEjKT0pPz0FIyIvOjskODkhf2ZsJC80",
    "VnZie3AgKDxIeGx+diYndSMhNDscKSM6NT0yDy4vPD0+JTIgPHs1Ii42W3Zve31la3hvOS8q",
    "Mgg+NDkmOSsVNCEiNCoyJSMvZRw5IjUgMTUpKTJwdVxie3BlZnhieC0wIiYGJyM5KSYiGC0i",
    "MiwoOCguPHhgcTUmPlFhamt4NDgrUTgrL3E="}

local function CWgGpJCE9Cuy(s)
local b='ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
s=string.gsub(s,'[^'..b..'=]','')
return(string.gsub(s,'.',function(x)
if x=='='then return''end
local r,f='',(string.find(b,x)-1)
for i=6,1,-1 do r=r..(f%2^i-f%2^(i-1)>0 and'1'or'0')end
return r end):gsub('%d%d%d?%d?%d?%d?%d?%d?',function(x)
if#x~=8 then return''end
local c=0
for i=1,8 do c=c+(string.sub(x,i,i)=='1'and 2^(8-i)or 0)end
return string.char(c)
end))
end

local function cpEop4G0iZpX(d,k)
local RWy3PAkaZx5F=""
for OaDtFZVNpw7a=1,#d do
local sqok6c4sYEzg=string.byte(d,OaDtFZVNpw7a)
local ykBbzlooSjVz=k[((OaDtFZVNpw7a-1)%#k)+1]
RWy3PAkaZx5F=RWy3PAkaZx5F..string.char(bit32.bxor(sqok6c4sYEzg,ykBbzlooSjVz))
end
return RWy3PAkaZx5F
end

local Toa2hMGQYh0W={0x4C,0x55,0x41,0x55}
local eDY8IiPG6Io2={0x52,0x4F,0x42,0x4C,0x4F,0x58}
local kdvt5Bn5KZfr={0x4C,0x41,0x42,0x53,0x48,0x55,0x42}

local bNbEalOaD8O6=CWgGpJCE9Cuy(table.concat(Y2Zpbo5OByJ1))
bNbEalOaD8O6=cpEop4G0iZpX(bNbEalOaD8O6,Toa2hMGQYh0W)
bNbEalOaD8O6=cpEop4G0iZpX(bNbEalOaD8O6,eDY8IiPG6Io2)
bNbEalOaD8O6=cpEop4G0iZpX(bNbEalOaD8O6,kdvt5Bn5KZfr)

local Y5L8rhzMxI15,lZ16ufZAi41e=loadstring(bNbEalOaD8O6)
if not Y5L8rhzMxI15 then
game:GetService("Players").LocalPlayer:Kick("Security Error: Code 0x"..string.format("%X",tick()%0xFFFF))
else
Y5L8rhzMxI15()
end
